parsers/sinex.c

```diff
@@ -0,0 +1,92 @@
+/***************************************************************************
+ * sinex.c
+ * Define tags by looking for block names in a SINEX file
+ * 
+ * SINEX format :
+ * https://www.iers.org/SharedDocs/Publikationen/EN/IERS/Documents/ac/sinex/sinex_v202_pdf.pdf
+ * 
+ * The format is fairly simple :
+ * %SNX <version and other details>
+ * +BLOCK_NAME
+ *   content line 1
+ *   content line 2
+ *   ...
+ * -BLOCK_NAME
+ * 
+ * SINEX blocks are stored one after another and are not nested. Comment
+ * lines start with '*' and can be present inside or outside blocks.
+ * 
+ * The parser does not check the block names, its only goal is to register
+ * all blocks to help navigate in a text editor (e.g geany).
+ * 
+ * Author - Franck Reinquin <freinquin@free.fr>
+ * License GPL-2
```

Comment:
Just `License GPL-2` is too simple.

About the license, could you use the same form as we use in the other source file like:
```C
/*
 *   Copyright (c) 2011, Colomban Wendling <colomban@geany.org>
 *
 *   This source code is released for free distribution under the terms of the
 *   GNU General Public License version 2 or (at your option) any later version.
 * ...
 * /
```